name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/MonthlyPNTransfersWidget.java

```diff
@@ -0,0 +1,68 @@
+package name.abuchen.portfolio.ui.views.dashboard;
+
+import java.time.LocalDate;
+import java.util.stream.LongStream;
+
+import name.abuchen.portfolio.model.Client;
+import name.abuchen.portfolio.model.Dashboard.Widget;
+import name.abuchen.portfolio.snapshot.PerformanceIndex;
+import name.abuchen.portfolio.ui.views.dashboard.heatmap.AbstractMonhtlyHeatmapWidget;
+import name.abuchen.portfolio.ui.views.dashboard.heatmap.HeatmapModel;
+import name.abuchen.portfolio.ui.views.dataseries.DataSeries;
+import name.abuchen.portfolio.util.Interval;
+
+public class MonthlyPNTransfersWidget extends AbstractMonhtlyHeatmapWidget
```

Comment:
Fixed typo: @buchen 
AbstractMonhtlyHeatmapWidget → AbstractMonthlyHeatmapWidget (missing 't' in Monthly)